packages/app/src/specs/SpecsListHoverCell.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div
+    ref="columnRef"
+    class="relative"
+  >
+    <slot
+      v-if="!shouldShowHover"
+      name="content"
+    />
+    <div
+      v-if="shouldShowHover"
```

Comment:
If we change this to a `v-else` we can wrap both of these in our `TransitionQuickFade` component which could be nicer than just popping these in.

Reply:
Done